backend/tests/test_b055_llm_worker_stubs.py

```
@@ -0,0 +1,189 @@
+        api_call = await session.get(LLMApiCall, UUID(result["api_call_id"]))
+        assert api_call is not None
+        assert api_call.tenant_id == test_tenant
+
+
+@pytest.mark.asyncio
+async def test_llm_monthly_costs_concurrent_updates_are_atomic(test_tenant):
+    _assert_postgres_engine()
+    increment = 7
+    calls = 8
+
+    async def _apply_increment():
```

Ruff (0.14.10)

Missing return type annotation for private function `_apply_increment`

Add return type annotation: `None`

(ANN202)